code/modules/admin/verbs/superheroes.dm

```diff
@@ -0,0 +1,58 @@
+
+/client/proc/summon_superheroes()
+	set category = "Admin.Fun"
+	set name = "Summon Superheroes and Supervillains"
+	set desc = "Spawn superhero and supervillain ships. USE ONLY ONCE OR SHIPS MAY BREAK."
+
+	var/choice = input(src, "Which teams do you want to spawn?") in list("Superheroes", "Supervillains", "Both superheroes and supervillains")
+
+	if(choice == "Superheroes" || choice == "Both superheroes and supervillains")
+		holder?.spawnSuperheroes()
+
+	if(choice == "Supervillains" || choice == "Both superheroes and supervillains")
+		holder?.spawnSupervillains()
+
+	var/list/candidates = pollGhostCandidates("Would you like to play as a superhero/supervillain?", ROLE_SUPERHERO, FALSE, 300)
```

Comment:
Would it be better to split this into two polls, in case someone wants to be a superhero but not a supervillain (or vice versa)?

Eh, who am I kidding, everyone would just say yes to both polls to have the highest possible chance to play a rare midround role (with either a license to kill or a license to validhunt) or say no to both polls if they're about to be rezzed or go eat dinner or something.

Reply:
if two polls show at once the first one is getting closed, so no.